src/coreclr/pal/src/arch/arm64/asmconstants.h

```diff
@@ -21,10 +21,8 @@
 #define CONTEXT_XSTATE_BIT (6)
 #define CONTEXT_XSTATE (1 << CONTEXT_XSTATE_BIT)
 
-#define XSTATE_SVE_BIT (0)
-
-#define XSTATE_MASK_SVE (UI64(1) << (XSTATE_SVE))
-
+#define XSTATE_ARM64_SVE_BIT (2)
```

Comment:
The general `CONTEXT` struct as is currently used by PAL is supposed to mirror the win32 layout/defines so they can translate 1-to-1

The win32 naming convention here is then `XSTATE_ARM64_SVE` and it is using bit 2, not bit 1

Reply:
> The win32 naming convention here is then XSTATE_ARM64_SVE and it is using bit 2, not bit 1

So wrong bit was set for linux in https://github.com/dotnet/runtime/pull/103801 as well?

Reply:
Right.

I don't think it would technically cause problems because we're just emulating the CONTEXT struct and it doesn't actually have to match the Win32 struct 1-to-1 (since its only going to be other pal APIs consuming it)

But it's overall better/simpler to ensure they match so we don't end up with potential conflicts or other issues.

The best option here would be to just consume the native `context_t` struct from Unix directly (and not do any of this shimming), but that's a more involved PR.

Reply:
> it doesn't actually have to match the Win32 struct 1-to-1

In fact, it does not match the Win32 struct 1-to-1: https://github.com/dotnet/runtime/pull/103801#discussion_r1649435005

> The best option here would be to just consume the native context_t struct from Unix directly (and not do any of this shimming), but that's a more involved PR.

+1. The Windows CONTEXT structure is very entrenched throughout CoreCLR VM.